.travis.yml

```diff
@@ -12,6 +12,10 @@ install:
 
 matrix:
   include:
+    - name: "presubmit"
+      install:
+        - pip install -r infra/dev-requirements.txt
```

Comment:
nit: Just requirements.txt is common name

Reply:
Yes but usually that's for the project itself. It's not clear from the path what requirements.txt is for so I'm naming it so that it is unambiguous.